src/utils/opcuaclient.js

```diff
@@ -109,13 +109,30 @@ const monitorVariable = async (subscription, item) => {
 
 const send = async (name, value) => {
   if (EGRESS_URLS) {
-    await fetch(EGRESS_URLS, {
-      method: 'POST',
-      body: JSON.stringify({
-        timestamp: Date.now(),
-        value,
-        name,
-      }),
+    const urls = []
+    const eUrls = EGRESS_URLS.replace(/ /g, '')
+    if (eUrls.indexOf(',') !== -1) {
+      urls.push(...eUrls.split(','))
+    } else {
+      urls.push(eUrls)
+    }
+    urls.forEach(async url => {
+      if (url) {
+        const callRes = await fetch(url, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            timestamp: Date.now(),
+            value,
+            name,
+          }),
+        })
+        if (!callRes.ok) {
+          console.error(`Error passing response data to ${url}`)
+        }
+      }
     })
   } else {
     console.log(
```

Comment:
is this the desired behavior? I think an error must be thrown in this case

Reply:
that is left for debugging in the case we want to see it in docker instance.

Reply:
but this goes into production, right?

Reply:
yes, but this goes to docker instance, we don't have a way to view logs from edge designer for the moment so we use here to check it out when debugging pipelines.

Reply:
but how throwing an error here is worse for debugging than just logging the data?

Reply:
I removed it, it was used in shell to debug nothing else since we don't have any kind of overview of module logs at the moment until we shell the instance.  removed it, now you can merge it.

Reply:
I think then we should also remove `if (EGRESS_URLS)`, because in case it's not set no error will be thrown and we'll be having a hard time finding the source of the error